src/pages/signin/SignInPage.js

```diff
@@ -49,15 +50,19 @@ const propTypes = {
     /** Whether or not the sign in page is being rendered in the RHP modal */
     isInModal: PropTypes.bool,
 
-    /** Override the green headline copy */
-    customHeadline: PropTypes.string,
+    /** Information about any currently running demos */
+    demoInfo: PropTypes.shape({
+        saastr: PropTypes.shape({
+            isBeginningDemo: PropTypes.bool,
+        }),
+    }),
```

Comment:
you're right. But this will come in a follow-up PR after we decide what wording we want to specific to SBE. In this PR we are mainly supporting Saastr for now.